pkg/tarmak/tarmak.go

```diff
@@ -350,6 +353,9 @@ func (t *Tarmak) Variables() map[string]interface{} {
 }
 
 func (t *Tarmak) Must(err error) {
+	t.Cleanup()
+	plugin.CleanupClients()
+
```

Comment:
This is the wrong method for this. `Must(error)` only handles a fatal error for tarmak. The convention is common in other libraries. Come up with a better method name and call out to the clean up methods and `Must(error)`